lib/core/network/api/eos_service.dart

```diff
@@ -15,55 +16,43 @@ class EOSService {
 
   EOSService(this.eosClient, this.secureStorageService);
 
-  EOSClient _withPrivateKey(String privateKey) {
-    eosClient.privateKeys = [privateKey];
-    return eosClient;
-  }
-
   Future<Result<dynamic>> sendTransaction({
     required EOSTransaction eosTransaction,
     required String accountName,
   }) async {
     final actions = eosTransaction.actions.map((e) => e.toEosAction).toList();
 
-    LogHelper.d('GERY GERY: sendTransaction ' + actions.toString());
     for (final action in actions) {
-      LogHelper.d('GERY GERY: sendTransaction Action: ' + action.toString());
       if (action.authorization == null || action.authorization == []) {
-        LogHelper.d('GERY GERY: sendTransaction Inside check: ');
         action.authorization = [
           Authorization()
             ..actor = accountName
             ..permission = 'active'
         ];
       }
     }
-    LogHelper.d('GERY GERY: sendTransaction Done wit if: ');
-
-    LogHelper.d('GERY GERY: sendTransaction Action: ' + actions.toString());
-
     final transaction = _buildTransaction(actions, accountName);
-
     final UserAuthData? userAuthData = await secureStorageService.getUserAuthData();
-    LogHelper.d('GERY GERY: ${userAuthData?.eOSPrivateKey?.toString()}');
 
-    return EOSClient(baseUrl: 'http://eos.greymass.com', privateKeys: ['NIK ENTER YOUR KEY'], version: 'v1')
```

Comment:
Intentionally creating a new eos client for this transaction - this way I use our push transaction node URL, different from the base URL

Overall I am not a big fan of creating a lot of static factory objects all over the place, if they are not really needed. And if they are created, I am not a fan of modifying them as a side effect of some call - yikes!